clients/include/blas1/testing_copy.hpp

```diff
@@ -36,6 +36,31 @@ inline void testname_copy(const Arguments& arg, std::string& name)
     hipblasCopyModel{}.test_name(arg, name);
 }
 
+template <typename T>
+void testing_copy_bad_arg(const Arguments& arg)
+{
+    bool FORTRAN       = arg.api == hipblas_client_api::FORTRAN;
+    auto hipblasCopyFn = FORTRAN ? hipblasCopy<T, true> : hipblasCopy<T, false>;
+
+    hipblasLocalHandle handle(arg);
+
+    int64_t N    = 100;
+    int64_t incx = 1;
+    int64_t incy = 1;
+
+    device_vector<T> dx(N * incx);
+    device_vector<T> dy(N * incy);
+
+    EXPECT_HIPBLAS_STATUS(hipblasCopyFn(nullptr, N, dx, incx, dy, incy),
+                          HIPBLAS_STATUS_NOT_INITIALIZED);
+#ifndef __HIP_PLATFORM_NVCC__
+    EXPECT_HIPBLAS_STATUS(hipblasCopyFn(handle, N, nullptr, incx, dy, incy),
+                          HIPBLAS_STATUS_INVALID_VALUE);
+    EXPECT_HIPBLAS_STATUS(hipblasCopyFn(handle, N, dx, incx, nullptr, incy),
+                          HIPBLAS_STATUS_INVALID_VALUE);
+#endif
```

Comment:
So how are these inconsistencies tracked?   Do we want to note them with some convention?

Reply:
Could maybe do it in the yaml files somehow, but might get messy. I'll see if I can think of something better.

Reply:
You can take a look at the last commit, it adds a flag to hipblas-arguments to enable "all" bad_arg tests. Not sure if it adds value or not, but essentially could just change .yaml files if we want to add these tests to cublas backend.